src/gen_dragonfruit/gd_dev_adafruit_MPL115A2_press.cpp

```diff
@@ -1,12 +1,37 @@
+/*******************************
+ *
+ * File: gd_dev_adafruit_MPL115A2_press.cpp
+ *
+ * This module is a driver for the pressure sensor. It uses the Adafruit 
```

Comment:
Techncialy this is not an adafruit sensor - adafruit creates the breakout board for this sensor that is actually manufactured by Freescale. 

For the drivers I would recommend adding a link to product page as well as the datasheet page.